### PR TITLE
Customizing GNJA SET - First attempt

### PR DESCRIPTION
**Pull request by Marittiell**
0xb788b752d205527b27caa5fe18230a74a1a264d358c96b4dd26c3816e43b9b0e4f8f70cd489f4b03af98a60f20625b738d5e0e5ec36fea4309902510f39c10311c

### CODE REVIEW

**Comment by Marittiell** [comment]
json modified in polygon set (right place....)
modified set image ratio

File: set-details/0x445f746925e2dC6A825d095e53984d034A4CFf39.setdetails.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "GainJA - the Gain Ninja",
+    "setSymbol": "GNJA",
+    "setDecimals": 18,
+    "setIcon": "https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/assets/tokens/GNJA.png",           
+    "setDescription": "Every day we try to optimize ratio among different tokens in order to gain a balanced exposure in every market. Our aim is a conservative strategy to simplify your asset management avoiding overexposure on both sides of the markets. Enjoy your life without following price action avery hour or day and remember that we care for you this bunch of cryptoassets.",
+    "website",
```
Comment (asoong) [inline set-details/0x445f746925e2dC6A825d095e53984d034A4CFf39.setdetails.json:9]: Need to provide a valid website here, can use empty quotes if necessary.

**Comment by asoong** [comment]
Bump here @Marittiell, you can leave the website empty, provide empty quotes.

File: set-details/0x445f746925e2dC6A825d095e53984d034A4CFf39.setdetails.json (hunk 1 above)
Comment (Marittiell) [reply set-details/0x445f746925e2dC6A825d095e53984d034A4CFf39.setdetails.json:9]: i used empty quotes

File: set-details/0x445f746925e2dC6A825d095e53984d034A4CFf39.setdetails.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "GainJA - the Gain Ninja",
```
Comment (justinkchen) [inline set-details/0x445f746925e2dC6A825d095e53984d034A4CFf39.setdetails.json:4]: is this set on both polygon and ethereum?

**Comment by justinkchen** [comment]
Closing due to inactivity

**Pull request closed without merging**